bin/ember-template-lint.js

```diff
@@ -131,10 +146,29 @@ function printPending(results, options) {
 function run() {
   let options = parseArgv(process.argv.slice(2));
   let positional = options._;
+  let config;
+
+  if (options.config) {
+    try {
+      config = JSON.parse(options.config);
+    } catch (error) {
+      console.error('Could not parse specified `--config` as JSON');
+      process.exitCode = 1;
+      return;
+    }
+  }
 
   let linter;
   try {
-    linter = new Linter({ configPath: options.configPath });
+    if (options['no-config-path'] !== undefined) {
+      options.configPath = false;
+    }
```

Comment:
This probably doesn't need to be in the `try`/`catch` block.

Reply:
moved out of the try catch block